### PR TITLE
[4.0] Change width to class

### PR DESCRIPTION
**Pull request by brianteeman**
Continuation of the work @quy did to replace style="width:xx%"

This time replaces width="xx%"

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 73be07481bf9bdd01837191c4a74df1233711f05<br /><br />Thank you! I forgot about the width attribute.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29122">issues.joomla.org/tracker/joomla-cms/29122</a>.</sub>

**Comment by richard67** [comment]
@brianteeman @Quy It seems for the installation, the "w-" classes are not known. E.g. in file `installation/tmpl/remove/default.php` I get a bigger width when using the unmodified core file than when using the file from this PR, and when using the file from this PR and inspecting the markup and css in browser tools, I see nowhere the width of n% applied.

Update: The rest works fine, I can see the width applied in broswer inspection tool. Only for the installation it is not the case.

**Comment by richard67** [comment]
So you can chose what is better: Keep the old width attribute in the installation, or add the "w-" classes to file `installation/template/scss/template.scss`.

**Comment by Quy** [comment]
It displays fine without `w-`. Let's remove them.

File: installation/tmpl/remove/default.php [hunk 1]
```diff
@@ -220,16 +220,16 @@ class="btn btn-primary"
 				<caption id="install_languages_desc"><?php echo Text::_('INSTL_LANGUAGES_DESC'); ?></caption>
 					<thead>
 					<tr>
-						<td width="1%" class="text-center">
+						<td class="text-center">
```
Comment (Quy) [inline installation/tmpl/remove/default.php]:
```suggestion
						<td>
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 36c9af2bc702da0fcba31d7fba6c03ab6f82d514<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29122">issues.joomla.org/tracker/joomla-cms/29122</a>.</sub>

**Comment by bonzani** [comment]
I have tested this item :white_check_mark: successfully on 36c9af2bc702da0fcba31d7fba6c03ab6f82d514<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29122">issues.joomla.org/tracker/joomla-cms/29122</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29122">issues.joomla.org/tracker/joomla-cms/29122</a>.</sub>

**Comment by richard67** [comment]
Back to pending. @brianteeman Could you fix PHPCS? See the log here [https://ci.joomla.org/joomla/joomla-cms/32237/1/9](https://ci.joomla.org/joomla/joomla-cms/32237/1/9 "https://ci.joomla.org/joomla/joomla-cms/32237/1/9").<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29122">issues.joomla.org/tracker/joomla-cms/29122</a>.</sub>

**Comment by brianteeman** [comment]
@richard67 that code is nothing to do with this PR

**Comment by brianteeman** [comment]
@richard67 they are fixed in #29163

**Comment by zero-24** [comment]
Merging here thanks @brianteeman

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**